app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= stylesheet_link_tag 'registrations/new', media: 'all' %>
+
+<div class='registeration-cont'>
+  <div class='sign-up-header'>
+ 
+<h2>Log In</h2>
+</div>
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: 'Email', class: 'form-inp' %>
+  </div>
+
+  <div class="field">
+    <%= f.password_field :password, autocomplete: "current-password",placeholder: 'Password', class: 'form-inp' %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me, class: 'form-checkbox'%>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log In", class: 'submit-btn' %>
+  </div>
+<% end %>
```

Comment:
- [ ] If you can see the login screen, the links are displayed in a weird way because the styles you did is for desktop screens and as soon as the view changes to mobile screens the texts are wrapped. Kindly improve the styles of the texts to display in one line 👍🏽

<img width="458" alt="Screenshot 2023-06-03 at 1 43 08 PM" src="https://github.com/The-Normal-One1/budget_app/assets/50941074/9bbb1eb1-9213-4be7-afc4-5f148f481b3a">